lib/lambda_ethereum_consensus/p2p/gossip/gossipsub.ex

```diff
@@ -43,6 +43,8 @@ defmodule LambdaEthereumConsensus.P2P.GossipSub do
         {Consumer, %{topic: topic, ssz_type: ssz_type, handler: handler}}
       end
 
+    children = [LambdaEthereumConsensus.P2P.Gossip.Attestation] ++ children
```

Comment:
This seems to be quite different to the way we were approaching gossip subscriptions? Is the idea to migrate all to a "separate genserver" format?

Reply:
I don't think so. In this particular case, we needed to dynamically join and leave topics, including subscribing to a particular subnet.

Reply:
But I imagine this would be the case for most if not all the topics? For example, when a fork changes then we need to join a new topic based on the new fork digest?

Reply:
(not in scope for this PR, just discussing)

Reply:
You're right. We'll need to support multiple topics when transitioning to a new hard-fork, but apart from that, there are some topics that remain static, like blocks.